test/cpp/qps/client_sync.cc

```diff
@@ -64,17 +66,70 @@ namespace testing {
 
 class SynchronousClient : public Client {
  public:
-  SynchronousClient(const ClientConfig& config) : Client(config) {
+  SynchronousClient(const ClientConfig& config) : Client(config),
+                                                  interarrival_timer_() {
     num_threads_ =
       config.outstanding_rpcs_per_channel() * config.client_channels();
     responses_.resize(num_threads_);
+
```

Comment:
Could this move into client.h (with, maybe, a ```c bool NextTimeout(timepoint*) ``` helper to get the next timeout (or not).